src/content/advanced/ingress/configuration/index.md

```diff
@@ -358,7 +362,7 @@ metadata:
   namespace: mynamespace
   annotations:
     nginx.ingress.kubernetes.io/configuration-snippet: |
-      expires 24h;
+   expires 24h;
```

Comment:
@giantswarm/sig-docs 

Seems to be related to https://github.com/alecthomas/chroma/issues/475

On page: https://docs.giantswarm.io/advanced/ingress/configuration/#configuration-snippets

![image](https://user-images.githubusercontent.com/1494211/158990341-e3061e66-ec01-4616-92b3-f8bbf4d082ba.png)


![image](https://user-images.githubusercontent.com/1494211/158990422-9d9d4f59-b743-4394-98f4-b7cb1cfdfe95.png)

Reply:
this is crazy